README.md

```diff
@@ -1369,7 +1381,7 @@ import itiriri from 'itiriri';
 itiriri([1, 2, 3]).skipWhile(() => true); // returns []
 itiriri([1, 2, 3]).skipWhile(() => false); // returns [1, 2, 3]
 itiriri([1, 2, 3]).skipWhile(e => e < 3); // returns [3]
-itiriri([1, 2, 3]).skipWhile(e => e % 2 === 0); // returns [1, 2, 3]
```

Comment:
`skipWhile` skips all the elements until the predicate is not satisfied. In this example the predicate is not satisfied at the very first example, and as a result the entire array is returned.